### ISSUE TITLE
Test multiple increment in incremental mode

### ISSUE THREAD

**Issue by fdelbrayelle** [fdelbrayelle/generator-jhipster-kafka#71]
##### **Overview of the issue**

It should have some tests in `app.spec.js` to test multiple loop increment in incremental mode.

##### **Motivation for or Use Case**

Currently there is only tests for one loop turn in incremental mode. This is configured like this in `app.spec.js`:

```
continueAddingEntitiesComponents: false
```

##### **Suggest a Fix**

Add some tests with:

```
continueAddingEntitiesComponents: true
```

And find a trick to loop in incremental mode in test scope.

##### **JHipster Version(s)**

6.9.0

- [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by monxxi** [comment]
yes, i already tried this approach, but i didn't manage to call the helpers withPrompts a second time in a callback provided in the `on()` 'method'.
